src/components/AccountCard/index.js

```diff
@@ -23,11 +23,14 @@ const AccountCard = ({ account }) => (
       </div>
       <div className="col-md-6 accountInfo">
         <p className="credibility">{account.credibility}<span className="uom">pts</span></p>
-        <p >Github: <a href={account.github_handle}>{account.github_handle}</a></p>
-        <p >LinkedIn: <a href={account.linkedin_url}>{account.linkedin_url}</a></p>
+        <p >Github: <a href={`https://github.com/${account.github_handle}`}>{account.github_handle}</a></p>
+        <p >LinkedIn: <a href={`https://www.linkedin.com/in/${account.linkedin_url}/`}>{account.linkedin_url}</a></p>
         <p >Resume Site: <a href={account.resume_site_url}>{account.resume_site_url}</a></p>
-        <p >StackOverflow: <a href={account.stackoverflow_url}>{account.stackoverflow_url}</a></p>
-        <p >Twitter: <a href={account.twitter_handle}>{account.twitter_handle}</a></p>
+        <p >StackOverflow: <a href={account.stackoverflow_url}>{account.stackoverflow_url
+          .substring(
+            account.stackoverflow_url.lastIndexOf('/') + 1,
```

Comment:
Hey @binarymason I added that to trim the username from the url so that the link displays as the stackoverflow username similar to the Twitter and Github links instead of `https://stackoverflow.com/users/<id>/<user_name>`. I can change it and resubmit if it's not inline with what you had in mind.